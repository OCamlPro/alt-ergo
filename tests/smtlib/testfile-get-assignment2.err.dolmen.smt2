(set-logic ALL)
(set-option :produce-models true)
(declare-const x Int)
(assert (! (= x 0) :named foo))
(check-sat)
(get-model)
; The next instruction should fail because we did not activate the option
(get-assignment)