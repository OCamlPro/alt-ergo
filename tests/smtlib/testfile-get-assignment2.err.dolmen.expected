
unknown
(
  (define-fun x () Int 0)
)
(error "Produce assignments disabled; add (set-option :produce-assignments true)")
